charts/posthog/templates/migrate.job.yaml

```
@@ -83,11 +83,11 @@ spec:
             {{- end }}
               key: {{ template "posthog.postgresql.secretKey" . }}
         - name: POSTHOG_POSTGRES_HOST
```

Worth adding a comment here since we have similar values elsewhere.

```suggestion
        # Connect directly to postgres (without pgbouncer) to avoid time limits
        - name: POSTHOG_POSTGRES_HOST
```